modules/sandboxed-containers-create-kata-config-resource-cli.adoc

```diff
@@ -60,7 +71,7 @@ spec:
 $ oc create -f <file name>.yaml
 ----
 
-The new `KataConfig` CR is created and begins to install `kata` as a `RuntimeClass` on the worker nodes.
+The new `KataConfig` CR is created and begins to install `kata` as a `RuntimeClass` on the worker nodes. Wait for the `KataConfig` CR creation to complete and the worker nodes to reboot before continuing to the next step.
```

Comment:
This is just a suggestion, the second sentence here is a little confusing because the CR "creation" could be interpreted to mean the _process of creating the CR_, which the customer is told is completed in the first sentence. Wdyt about rewording to something like the following?
```suggestion
The new `KataConfig` CR is created and begins to install `kata` as a `RuntimeClass` on the worker nodes. Wait for the installation to complete and the worker nodes to reboot before continuing to the next step.
```

Reply:
Done.